autoload/lsp/lsp.vim

```diff
@@ -252,14 +253,21 @@ def BufferInit(bnr: number): void
   # add a listener to track changes to this buffer
   listener_add(Bufchange_listener, bnr)
 
+  # FIXME: Maybe we should introduce defaults regardless of auto/omnicompl
```

Comment:
Can you remove this change from this PR?  If needed, we can make this change in a separate PR.

Reply:
sure, understandable - get the history right.

I'll open a PR afterwards.